src/src/Helpers/StringParser.cpp

```diff
@@ -56,12 +70,13 @@ String parseTemplate_padded(String& tmpString, uint8_t minimal_lineSize, bool us
   if (parseTemplate_CallBack_ptr != nullptr) {
     parseTemplate_CallBack_ptr(tmpString, useURLencode);
   }
-  parseSystemVariables(tmpString, useURLencode);
-
+  if (!hasEscapeCharacters(&tmpString, "\\%"))
+    parseSystemVariables(tmpString, useURLencode);
 
   int startpos = 0;
   int lastStartpos = 0;
   int endpos = 0;
+  if (!hasEscapeCharacters(&tmpString, "\\[") && (!hasEscapeCharacters(&tmpString, "\\]")))

```

Comment:
So if there are escape characters present, the entire template won't be parsed?
That's not how it should go, IMHO.
- `parseSystemVariables()` and `parseStandardConversions()` should each take care of the `\%` escape itself, like the next description:
- `parseTemplate_padded()` should:
  - first replace each escape to be excluded for the function (`\[` and `\]`) by f.e. an ASCII character below 0x20, like 0x04 and 0x05 (AFAIK, 0x01 .. 0x03 are being used somewhere already, have to check).
  - The process then should replace all (remaining) variables as usual
  - Before returning the result, the replacements should be reverted.

(Still checking if there are more functions to adjust)